tests/jobs/scala/src/main/scala/sorting/DatasetGenerator.scala

```diff
@@ -10,6 +11,13 @@ import scala.util.Random
 
 case class DataItem(key: String, value: Array[Byte])
 
+object DataItem {
+	def schema = StructType(Seq(
+		StructField("key", StringType, nullable = false),
+		StructField("value", DataTypes.BinaryType, nullable = false)
+	))
+}
+
 case class Config(numFiles: Int = -1,
 									numRecords: Int = -1,
 									staticPrefixLength: Int = 120,
```

Comment:
Not really part of your PR but can we fix this wild formatting? Not sure why all of these params are so far away. (future idea: might be good to add a linter for the Scala code to avoid these issues)

Reply:
wild formatting fixed here and everywhere